src/Renderer/ThreeExtented/GlobeControls.js

```diff
@@ -1179,9 +1144,13 @@ GlobeControls.prototype.setHeading = function(heading) {
     this.updateCameraTransformation();
 };
 
+GlobeControls.prototype.getCenter = function() {
```

Comment:
`getCenter`: center of what? center of view? on the globe?
Add a single line of comment or rename using a more descriptive  name